assets/app/view/game/player.rb

```diff
@@ -101,6 +101,13 @@ def render_info
               h('td.right', @game.format_currency(@player.cash - committed)),
             ]),
           ]) if committed.positive?
+
+          trs.concat([
+             h(:tr, [
+               h(:td, 'Bidding tokens'),
+               h('td.right', "#{@game.active_step.bidding_tokens(@player)} / #{@game.bidding_token_per_player}"),
+             ]),
+           ]) if @game.active_step&.current_actions&.include?('bidding_tokens')
```

Comment:
This is the only other place using the bidding_tokens action. Change to `@game.active_step.respond_to?(:bidding_tokens)`, so we don't need a special action.